config/api.yml

```diff
@@ -2800,6 +2804,10 @@
         :identifier: catalogitem_delete
       - :name: order
         :identifier: svc_catalog_provision
+      - :name: archive
+        :identifier: svc_catalog_archive
+      - :name: unarchive
+        :identifier: svc_catalog_unarchive
```

Comment:
the other failures might be because svc_catalog_archive and svc_catalog_unarchive are not mid_product_feature.yml

Reply:
Think this was actually related to https://github.com/ManageIQ/manageiq-api/pull/389#discussion_r193052475 since I had https://github.com/ManageIQ/manageiq/pull/17518 applied locally when testing and this still failed.